draft-thomson-webpush-protocol.xml

```diff
@@ -236,41 +230,37 @@
 ]]></artwork>
       </figure>
       <t>
-        At the very beginning of the process, the user agent creates a
-        registration resource at the push service.  The registration is the
-        basis of all future interactions between the user agent and push
-        service.  The registration aggregates push messages that the user agent
-        receives from all subscriptions.
-      </t>
-      <t>
-        The registration resource describes how the user agent is expected to
-        monitor for incoming push messages.  This document describes one such
-        mechanism, though more efficient means of monitoring could be defined.
+        At the very beginning of the process, a new subscription is created by
+        the user agent and then distributed to its application server. The
+        subscription is the basis of all future interactions between the actors.
       </t>
       <t>
-        The registration resource also describes how the user agent might create
-        a subscription.  A new subscription is created by the user agent and
-        then distributed by to an application server.
+        A subscription is modeled as two resources:
+        <list style="symbols">
+          <t>
+            A "monitor" resource grants read access to a subscription. User
+            agents use monitor resources to receive incoming push messages.
```

Comment:
We might also note that it offers control options here.  I'm inclined to talk about this as the "subscription" resource now.  The "monitor" verb is just one of the things you can do with the URL.